set.go

```diff
@@ -832,7 +832,7 @@ func (cc *Conn) GetSets(t *Table) ([]*Set, error) {
 
 	reply, err := receiveAckAware(conn, message.Header.Flags)
 	if err != nil {
-		return nil, fmt.Errorf("Receive: %v", err)
+		return nil, fmt.Errorf("receiveAckAware: %v", err)
```

Comment:
Would you mind sending these unrelated fixes as a separate PR? I’d like to keep the refactoring as self-contained and minimal as possible.

Reply:
Done. Will be addressed separately.